app/views/helpers/TimelineEvents.scala

```diff
@@ -115,15 +130,20 @@ class TimelineEvents @Inject() (
 
   private def isIndex1Primary(index1: Int, index2: Int): Boolean = index2 < 0 || index1 < index2
 
-  private def fixAndResubmitContent(id: String, isAmendmentRequest: Boolean)(implicit messages: Messages): Html = {
+  private def fixAndResubmitContent(id: String, maybeAmendmentToHandleAsLatest: Option[AmendmentAsLatest])(implicit messages: Messages): Html = {
```

Comment:
And `maybeAmendmentToHandleAsLatest` could be `amendmentOutcome` too?
`maybeAmendmentToHandleAsLatest` seems a bit obscure.